pkg/my/domain.go

```diff
@@ -18,40 +21,51 @@ type Domains struct {
 	domains []Domain
 }
 
-func (d *Domains) List() []Domain {
-	return d.domains
-}
-
-func (d *Domains) Save() error {
-	cnt, err := json.Marshal(d.domains)
+func GetDomains(client *node.Client, nickname string) ([]string, error) {
```

Comment:
Usage will be like : 
```go
...
my.GetDomains(...)
...
```
I porpose to change the function name to Domains to get :
```go
...
my.Domains(...)
...
```

Note: in Go, we avoid using Get prefix.